lib/Routing/Redirect/Csv.php

```diff
@@ -77,7 +78,12 @@ public function createExportWriter(Redirect\Listing $list): Writer
                 $document = Document::getById((int)$redirect->getTarget());
 
                 if ($document) {
-                    $target = $document->getRealFullPath();
+                    $targetSiteId = $redirect->getTargetSite();
+                    $targetSite = Site::getById($targetSiteId);
+                    $rootDocument = $targetSite->getRootDocument();
```

Comment:
this will break csv export when exporting main site entries where targetSite is null